IoTBlockly/ChakraHost/ChakraHostAsync.cs

```diff
@@ -90,109 +96,69 @@ public string dispose()
             return "";
         }
 
-        public void runScript(string script)
+
+        public void ScriptProcessor()
```

Comment:
I like the new ScriptProcessor(). Few comments:
* minor: let's use the same "scriptProcessor()" naming convention to be consistent
* where do we call init()? I don't see a call to init() anymore...